content/02.delete-me.md

```diff
@@ -65,34 +60,56 @@ two spaces at end of line
 3. Ordered list item
     a. Sub-item
 
-* List item
 - List item
-+ List item
+- List item
+- List item
 
 superscript<sup>2</sup>
 
 subscript<sub>2n+1</sub>
 
-[Link](https://www.google.com)
+unicode superscript™
+
+unicode subscriptₓ
+
+**Links and Citations**
+
+Bare URL link: <https://manubot.org>
 
-[Link with title (hover) text](https://www.google.com "Google")
+[Link with text](https://www.manubot.org)
 
-[Link by reference][google homepage]
+[Link with hover text](https://www.manubot.org "Manubot Homepage")
 
-[Google Homepage]: https://www.google.com
+[Link styled as a button](https://www.manubot.org "Manubot Homepage"){.button}
 
-[Link styled as a button](https://www.google.com "Google"){.button}
+[Link by reference][manubot homepage]
 
-DOI citation [@doi:10.15363/thinklab.4]
+[Manubot Homepage]: https://www.manubot.org
 
-URL citation [@url:https://greenelab.github.io/meta-review]
+Citation via DOI: [@doi:10.15363/thinklab.4]
 
-Citation by tag [@tag:deep-review]
+Citation by tag: [@tag:deep-review]
 
-> Quoted text 
+Manual URL citation: [@url:https://greenelab.github.io/meta-review]
```

Comment:
This is an automated URL citation.  We can keep it, but I was thinking we should still include an example of a manual override for some reference.  Specifically, updating https://github.com/manubot/rootstock/blob/master/content/manual-references.json and citing that `standard_citation` in this file.

GitHub URLs are often good candidates because the automatic citation cannot extract reasonable authors or titles.  Should I make an edit that uses `https://github.com/manubot/rootstock` as the example manual reference?

Reply:
We should add

```
Multiple citations together [@doi:10.15363/thinklab.4; @tag:deep-review; @url:https://greenelab.github.io/meta-review].
```